Pages/Shared/_Layout.cshtml.css

```diff
@@ -11,7 +11,7 @@ a {
   color: #0077cc;
 }
 
-.btn-primary {
+.button-primary {
```

Comment:
Consistent with naming conventions for button styling.

Reply:
Agreed.  Generally, it's good to not use abbreviations.  While it feels like you save time typing a longer variable name, you actually lose much more time when you forget what abbreviation you used, or you use abbreviations inconsistently.